_includes/extensions-form.md

```diff
@@ -0,0 +1,2 @@
+> Already built an extension? [Let us know about your experience](https://docs.docker.com/feedback/extension/).
+{: .important}
```

Comment:
(Not introduced in this PR) we should have a look at the link-colour for the "important" styles (blue on red looks a bit .. meh)

<img width="901" alt="Screenshot 2023-01-09 at 15 01 47" src="https://user-images.githubusercontent.com/1804568/211325926-6daaa3ca-4a5a-4393-8a91-53a10f478d97.png">


Haven't tried what it'd look like, but wondering if it looks better if we split the first part to a separate line, so that it'd become a heading in the block;


```suggestion
> Already built an extension?
>
> [Let us know about your experience](https://docs.docker.com/feedback/extension/).
{: .important}
```